### DISCUSSION TITLE
create-svelte package with an option "with Tailwind"

### DISCUSSION THREAD

**Discussion by mkv27** [sveltejs/kit#3025] [Ideas]
Based on this:
https://twitter.com/adamwathan/status/1467300908270862341?s=20

Scripts example (+typescript+prettier+eslint+tailwind):
```json
{
...
"scripts": {
    "dev": "concurrently \"pnpm run dev:css\" \"svelte-kit dev\"",
    "build": "pnpm run build:css && svelte-kit build",
    "package": "svelte-kit package",
    "preview": "svelte-kit preview",
    "check": "svelte-check --tsconfig ./tsconfig.json",
    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
    "lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
    "format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. .",
    "build:css": "NODE_ENV=production tailwindcss -o src/tailwind.css -c tailwind.config.cjs -m",
    "dev:css": "tailwindcss -o src/tailwind.css -c tailwind.config.cjs --watch"
  }
...
}
```

**Comment by bluwy** [comment]
There's https://github.com/svelte-add/tailwindcss that handles this. I think this different setup should reside there as an option instead. SvelteKit also recommends it for third-party integrations.

**Comment by dominikg** [comment]
i don't think using a separate process to generate css is a good idea. postcss itself isn't the fastest and doing it twice in parallel won't make it better.
Also if you want to use tailwind directives in .svelte file style blocks that won't work
